longevity_sla_test.py

```diff
@@ -26,10 +26,12 @@ class LongevitySlaTest(LongevityTest):
     STRESS_ROLE_NAME_TEMPLATE = 'role%d_%d'
     STRESS_ROLE_PASSWORD_TEMPLATE = 'rolep%d'
     SERVICE_LEVEL_NAME_TEMPLATE = 'sl%d_%d'
+    FULLSCAN_SERVICE_LEVEL_SHARES = 600
```

Comment:
Don't we want to be able to configure this value ?

Reply:
This value has not matter. We just want to run full scan with some service level.